.jenkins/common.groovy

```diff
@@ -22,7 +22,7 @@ def runTestCommand (platform, project)
     def command = """#!/usr/bin/env bash
                 set -x
                 cd ${project.paths.project_build_prefix}/build/release/test
-                NCCL_DEBUG=INFO HSA_FORCE_FINE_GRAIN_PCIE=1 ./UnitTests --gtest_output=xml --gtest_color=yes
+                {sudo} NCCL_DEBUG=INFO HSA_FORCE_FINE_GRAIN_PCIE=1 ./UnitTests --gtest_output=xml --gtest_color=yes
```

Comment:
We don't need HSA_FORCE_FINE_GRAIN_PCIE=1 anymore

Reply:
I thought @wenkaidu said it could be needed on PCIe systems.

Reply:
Yes, this is still needed on PCIe machines